modules/cnf-configure_for_irq_dynamic_load_balancing.adoc

```diff
@@ -0,0 +1,156 @@
+[id="configuring_for_irq_dynamic_load_balancing_{context}"]
+= Configuring a node for IRQ dynamic load balancing
+
+To configure a cluster node to handle IRQ dynamic load balancing, do the following:
+
+. Log in to the {product-title} cluster as a user with cluster-admin privileges.
+. Set the performance profile to use `apiVersion: performance.openshift.io/v2`.
+. Remove the `GloballyDisableIrqLoadBalancing` field or set it to `false`.
+. Set the appropriate isolated and reserved CPUs. The following snippet illustrates a profile that reserves 2 CPUs:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: dynamic-irq-profile
+spec:
+  cpu:
+    isolated: 2-5
+    reserved: 0-1
+----
+. Create the pod that uses exclusive CPUs, and set `irq-load-balancing.crio.io` and `cpu-quota.crio.io` annotations to `disable`. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dynamic-irq-pod
+  annotations:
+     irq-load-balancing.crio.io: "disable"
+     cpu-quota.crio.io: "disable"
+spec:
+  containers:
+  - name: dynamic-irq-pod
+    image: "quay.io/openshift-kni/cnf-tests:4.6"
+    command: ["sleep", "10h"]
+    resources:
+      requests:
+        cpu: 2
+        memory: "200M"
+      limits:
+        cpu: 2
+        memory: "200M"
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: ""
+  runtimeClassName: dynamic-irq-profile
```

Comment:
The runtimeclass name should be: performance-<profile_name> , so performance-dynamic-irq-profile in this case (I have the same bug in the upstream doc, I will fix it too)